ArraysAmbiguousSNPMiscallingMar2019/allele_flip/README.md

```diff
@@ -1,27 +1,57 @@
 
 **Invocations:**
 
-You can calculate the allele frequencies of your genotypes file by running `plink --freq`. Remember to add `--keep-allele-order` to forces the original A1/A2 allele encoding to be preserved.
+```
+usage: Rscript compare_frequencies_against_1kg.r \
+  --frq <PLINK_FRK_FILE> \
+  --frq1kg <1KG_FRK_FILE> \
+  [--pop <POP>] \
+  [--out <OUT>]
+                       
+<PLINK_FRK_FILE> = allele frequencies calculated using `PLINK --freq` command.
+<1KG_FRK_FILE> = allele frequencies of 1000 genomes from ./1000_Genomes_AF subdirectory
+<POP> = population name choosing from EUR, EAS and AFR. default is EUR
+<OUT> = output plot name. default is `af.pdf`
+```
+
+To run this R script, package `dplyr`, `ggplot2` and `optparse` should be installed.
+
+**Examples:**
+
+You can calculate the allele frequencies of your genotypes file by running `plink --freq`. Remember to add `--keep-allele-order` to forces the original A1/A2 allele encoding to be preserved. Both [PLINK 1.07](http://zzz.bwh.harvard.edu/plink/) and [1.9](https://www.cog-genomics.org/plink2) support these commands. 
 
 ```
-plink --bfile <PLINK BFILE> --freq --keep-allele-order
+plink --bfile <BFILE> --freq --keep-allele-order
 ```
 
-Then you can evaluate whether or not your data is affected by running the following command (suppose your data is from European ancestry, and the genotyping platform is GSAMD). 
+Running the command above will generate a `plink.frq` file. Then you can evaluate whether or not your data is affected by running the command below. In this example, your data is of European ancestry, and the genotyping platform is GSAMD. If not output filename is specified, a `af.pdf` file whill be generated in the current directory. 
 
 ```
 Rscript compare_frequencies_against_1kg.r \
  --frq plink.frq \
- --frq1kg 1000_Genomes_AF/GSAMD-24v1-0_20011747_A1.1.3.1000_Genomes_AF.txt
+ --frq1kg 1000_Genomes_AF/GSAMD-24v1-0_20011747_A1.1.3.1000_Genomes_AF.txt \
+ --pop EUR
+```
+Suppose your genotyping platform used is PsychChip_v1-1, the ancestry of your cohort is East Asian, and you want your output file to be named as `af_eas.pdf`. You should run: 
+
+```
+Rscript compare_frequencies_against_1kg.r \
+ --frq plink.frq \
+ --frq1kg 1000_Genomes_AF/PsychChip_v1-1_15073391_A1.1.3.1000_Genomes_AF.txt \
+ --pop EAS \
+ --out af_eas.pdf 
 ```
 
 
-## Flipping Affected Varaints 
-The python script `allele_flip.py` can take the PLINK bim file and inteval list file as inputs to flip the DNA strand of affected variants (swaps A↔T and C↔G)
+## Flipping Affected SNPs 
+The python script `allele_flip.py` can take the PLINK .bim file and interval list file as inputs to flip the alleles of affected SNPs (i.e. swaps A↔T and C↔G alleles, but does not flip strand). If not log filename is specified, a default `allele_flip.log` file will be generated.`
```

Comment:
it seems that the script was changed, so that in the absence of a specified log file, the log will be output to stdout....please fix the comment.

Reply:
fixed!